interfaces/dotnet/Cantera.Tests/src/EnumsTest.cs

```diff
@@ -0,0 +1,67 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace Cantera.Tests;
+
+public class EnumsTests
+{
+    [Fact]
+    public void ThermoPair_ToStringsCorrently()
```

Comment:
Is this supposed to be
```suggestion
    public void ThermoPair_ToStringsCorrectly()
```
?